vendor/bat-native-ads/src/bat/ads/internal/processors/contextual/text_embedding/text_embedding_event_unittest_util.cc

```diff
@@ -0,0 +1,25 @@
+/* Copyright (c) 2021 The Brave Authors. All rights reserved.
```

Comment:
`2022`. If you could please change any other new files as they should also be 2022.